src/controllers/submitUrlController.js

```diff
@@ -76,10 +77,11 @@ class SubmitUrlController extends UploadController {
     return url.length <= 2048
   }
 
-  static async getHeadRequest (url) {
+  static async headRequest (url) {
     try {
-      return await axios.head(url)
+      return await axios.head(url, { headers: { 'User-Agent': 'check service' } })
```

Comment:
Is there a reason why we're overriding the `User-Agent` on here? Is this needed for the API to work?

Reply:
That's the point of this PR - see my comments here https://github.com/digital-land/submit/issues/183#issuecomment-2317143767